### PR TITLE
modal with slider demo in /text_components/, button 'Launch Trigger & Textkomponent modal'

### PR DESCRIPTION
**Pull request by roschaefer**
@jtackmann have a look

You can see the slider below the form at `/triggers/:id/edit'

### CODE REVIEW

**Comment by roschaefer** [comment]
Oh, and you have to run `bundle install` in order to get the `bootstrap-slider-rails` gem.

**Comment by drjakob** [comment]
(I can't see it.)

**Comment by roschaefer** [comment]
@drjakob it's not deployed. I've talked to @jtackmann that I will push a pull request so he doesn't have to bother about how to integrate the gem.

**Comment by roschaefer** [comment]
@drjakob wrong comment - of course it's `/triggers/:id/edit`

**Comment by drjakob** [comment]
There is it. 200!

**Comment by jtackmann** [comment]
dear @drjakob and @roschaefer,
the slider demo is placed now in a modal with static content, that is accessible on /text_components via the button 'Launch Trigger & Textkomponent modal'.

**Comment by drjakob** [comment]
Thanks, feels much better. (Sometimes Sliders are not shown first time I load page. Explanation? Google Chrome on OS-X)

**Comment by drjakob** [comment]
@jtackmann We have to find a easy to use solution selecting multiple events. There will be +200 Events. (http://sensorstory.com/events) I proposed earlier to sort them in groups: https://github.com/roschaefer/story.board/issues/173

**Comment by roschaefer** [comment]
@drjakob page reload shouldn't be necessary now

File: app/views/text_components/index.html.haml [hunk 1]
```diff
@@ -38,3 +38,174 @@
...
+                      %label.checkbox-inline
+                        %input#inlineCheckbox2{:type => "checkbox", :value => "option2"}/
+                        %span.checkbox-button no
+                  .col-md-6
+                    .form-group
+                      %label Triggercolor
+
+                .row
+                  .col-md-12
+                    = render 'form'
```
Comment (roschaefer) [inline app/views/text_components/index.html.haml]: @jtackmann looks good.
Forms are shared across views in Rails. Look at my last commit. I reused `render 'form'`, so the same HTML will be generated in the modal that is generated in the `/text_components/new` and `/text_components/edit` view. For new views, like your modal, you just have to add instance variables like `@text_component = ...` in the controller. You don't have to do that for existing views (makes it easier).

Comment (jtackmann) [inline app/views/text_components/index.html.haml]: @roschaefer perfect, thanks for changing and letting me know!

**Comment by jtackmann** [comment]
@roschaefer @drjakob the modal dummy content includes now all sliders with additional javascript to show initially tooltips with custom formats (eg: 24h time format).

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -16,3 +16,82 @@
 //= require turbolinks
```
Comment (jtackmann) [inline app/assets/javascripts/application.js]: by mistake, took it off again

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -16,3 +16,82 @@
...
+
+jQuery(document).ready(function($) {
+
+
+	var sliderTemperature = $(".slider-temperature").slider({
+		formatter: function(value) {
+			return value + ' °C';
+		}	
+	});
+	sliderTemperature.slider('setValue',[33,66]);
```
Comment (jtackmann) [inline app/assets/javascripts/application.js]: can't really handle those js failures

**Comment by jtackmann** [comment]
hi @drjakob, hi @roschaefer,
i added the missing pieces to the modal. hope it fits for the presentation like this. let me if something urgent is left out.

**Pull request merged**